tests/operator_test.go

```diff
@@ -2213,6 +2214,62 @@ spec:
 			errMsg := fmt.Sprintf("Unsupported value: \"%s\"", incorrectOperator)
 			patchKvWorkloads(&incorrectWorkload, true, errMsg)
 		})
+
+		It("[test_id:5758]should check if all nodes have correct os label", func() {
+			nodes, err := virtClient.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(nodes.Items).NotTo(BeEmpty())
+			for _, n := range nodes.Items {
+				if _, ok := n.Labels[k8sv1.LabelOSStable]; !ok {
+					if _, ok := n.Labels["beta.kubernetes.io/os"]; !ok {
+						Expect(n.Labels[k8sv1.LabelOSStable]).ToNot(BeEmpty(), fmt.Sprintf("node doesn't have  %s label", k8sv1.LabelOSStable))
+					}
+				}
+			}
+		})
+
+		It("[test_id:5760]should schedule vm on windows node", func() {
```

Comment:
Do you expect that it gets scheduled, or do you expect that it stays in unscheduled state?